cellprofiler/modules/gaussianfilter.py

```diff
@@ -36,7 +36,7 @@ def create_settings(self):
 
         self.sigma = cellprofiler.setting.Integer(
             text="Sigma",
-            value=1
+            value=1, doc="Standard deviation of the kernel to be used for blurring. Larger sigmas induce more blurring."
```

Comment:
Going in line with PEP8 standards the `doc` keyword should probably be on a new line here